### PR TITLE
Fix: Remove Text autocomplete from test create form #4002

### PR DESCRIPTION
**Pull request by Pravesh-Sudha**
This PR...

## Changes

-

## Fixes

- https://github.com/kubeshop/testkube/issues/4002

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/pages/Tests/TestsList/TestCreationModalContent/TestCreationForm.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ const TestCreationForm: React.FC<TestCreationFormProps> = props => {
           rules={[required, k8sResourceNamePattern, k8sResourceNameMaxLength]}
           required
         >
-          <Input placeholder="e.g.: my-test" />
+          <Input placeholder="e.g.: my-test" autoComplete="off"/>
```
Comment (rangoo94) [inline src/components/pages/Tests/TestsList/TestCreationModalContent/TestCreationForm.tsx:116]: Did you test it in multiple browsers? As I wrote in https://github.com/kubeshop/testkube/issues/400, `autoComplete` with an unknown value (i.e. `nope`) works more often. The implementation is changing over time though, so I'm not 100% sure if it's best at this very moment.

```suggestion
          <Input placeholder="e.g.: my-test" autoComplete="nope" aria-autocomplete="none" />
```

**Review by rangoo94 — COMMENTED** [review]
It's worth adding it to other "name" fields as well.

File: src/components/pages/Tests/TestsList/TestCreationModalContent/TestCreationForm.tsx (hunk 1 above)
Comment (Pravesh-Sudha) [reply src/components/pages/Tests/TestsList/TestCreationModalContent/TestCreationForm.tsx:116]: It worked like 7-8 times in Edge and chrome, I test it with autocomplete to nope

**Comment by rangoo94** [comment]
Hi @Pravesh-Sudha, thanks for the PR! Can you add it to the `name` fields of the other resource types, per the ticket comment?

Besides, @pavloburchak, if you would have some time, please check if it works in Chrome/Firefox/Safari. Otherwise, I'll try to check it about the end of next week 🙂 

### Test Scenario

* Open form
* Put some name
* Submit form
* Open form
* Put a similar name
* Check if there is autocompletion during that

The scenario should be repeated twice - once submitting with Enter key, and once submitting with the Submit button.

**Comment by Pravesh-Sudha** [comment]
Sorry to sound noob, I am new to typescript, I have added the turn off autocomplete in the name field of TestSuiteCreationModelContent

**Comment by rangoo94** [comment]
Thank you @Pravesh-Sudha for your efforts! Looks like we have updated it in the whole application with https://github.com/kubeshop/testkube-dashboard/pull/889 PR, so I'll close this one.

**Pull request closed without merging**